### PR TITLE
Landing page & Docs for Alert component

### PR DESCRIPTION
**Pull request by prajapati-priyanka**
-add a landing page.
-add documentation for the alert component.
-components in the side-nav are redirected to the alert page itself.
- add links to every component once I make it.
- sidenav and header are not fixed. I will do it in the end.

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
Good to merge

File: component/alert.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  <link rel="stylesheet" href="/style.css" />
+  <link rel="stylesheet" href="/component/alert.css" />
+
+  <title>PineUI | Alert</title>
+</head>
+
+<body>
+  
+    <header class="header">
+      <figure class="logo-container">
```
Comment (VishalPatil18) [inline component/alert.html]: Great use of Semantic HTML 👍

File: component/alert.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+         <li><a href="/component/alert.html">Docs</a></li>
+         <li><a href="https://github.com/prajapati-priyanka">Github</a></li>
+        </ul>
+      </div>
+    </header>
+    <main class="component-container wrapper">
+      <aside class="sidenav">
+        <h3 class="sidenav-heading">Components</h3>
+        <ul>
+          <li><a class=" active md-text" href="/component/alert.html">Alert</a></li>
```
Comment (VishalPatil18) [inline component/alert.html]: Empty space before `active` could be avoided.

File: component/alert.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            The alert component has four variants:
+            <span class="highlight">alert-danger</span>,
+            <span class="highlight">alert-success</span>,
+            <span class="highlight">alert-info</span>
+            <span class="highlight">alert-warning</span>
+            
+          </p>
+          <div class="component-code lg-text">
+            <p class="alert alert-danger">
+              <strong>Danger!</strong> Indicates a dangerous or potentially
```
Comment (VishalPatil18) [inline component/alert.html]: Great use of `strong` 👍

File: style.css [hunk 4]
```diff
@@ -131,8 +131,92 @@ a {
   text-decoration: underline;
 }
 
+/* wrapper for side navbar and component display section  */
+.wrapper {
+  width: 90%;
+  height: 100vh;
+  margin: 0 auto;
+
```
Comment (VishalPatil18) [inline style.css:139]: You could avoid an empty line here also and at multiple occurrences in this same file.

File: style.css [hunk 5]
```diff
@@ -131,8 +131,92 @@ a {
...
+}
+
+.component-container {
+  display: flex;
+}
+
+ /* Side Navbar */
+
+ 
+
```
Comment (VishalPatil18) [inline style.css:149]: Multiple empty lines could be avoided as that breaks the flow of the code reader/reviewer.

File: style.css [hunk 6]
```diff
@@ -131,8 +131,92 @@ a {
...
+  color: var(--secondary-color);
+  font-weight: var(--bold-font);
+}
+
+ /* component display section  */
+
+.comp-display-section {
+  width: 70%;
+  margin: 0 auto;
+  margin-top: 0.8rem;
```
Comment (VishalPatil18) [inline style.css:181]: You could use the 3 valued shorthand for this purpose as it does the same work in less lines.

`margin: 0.8rem auto 0;` 

which works in this way
`margin: [margin-top] [margin-left-right] [margin-bottom];`

The third demo in [this](https://developer.mozilla.org/en-US/docs/Web/CSS/margin) might help visualize

**Review by VishalPatil18 — APPROVED** [review]
- A few suggestions regarding empty lines were added
- A suggestion regarding shorthand for margin
- The navbar and sidebar in your documentation page can be made fixed in the future, as that helps in better UX.

Apart from these the code and UI are quite clean and work great. Also great use of Semantic HTML. Well done @prajapati-priyanka 👍

File: component/alert.html (hunk 1 above)
Comment (prajapati-priyanka) [reply component/alert.html]: -Yes, I will fix my header and side navbar. I tried it but it was creating a problem so thought of doing it at the end.
-Regarding empty lines, do I have to do it manually or prettier does it automatically?

**Review by kart-c — APPROVED** [review]
Looks good

File: component/alert.html (hunk 2 above)
Comment (prajapati-priyanka) [reply component/alert.html]: okay, I will do that.

File: style.css (hunk 6 above)
Comment (prajapati-priyanka) [reply style.css:181]: Thank you for this explanation. I am gonna use this in the future.

File: style.css (hunk 4 above)
Comment (prajapati-priyanka) [reply style.css:139]: okay.

File: component/alert.html (hunk 1 above)
Comment (VishalPatil18) [reply component/alert.html]: I'm not sure but I don't think prettier does that. As far as I know prettier only breaks the `<i>` tag inside an `<a>` tag.

**Pull request merged**